docker_push

```diff
@@ -1,3 +1,17 @@
 #!/bin/bash
-docker login -u "$DOCKER_USER" -p "$DOCKER_PASS";
-docker push budgetkey/budgetkey-app-generic-item:latest
+if [ "${DOCKER_USERNAME}" != "" ] && [ "${DOCKER_PASSWORD}" != "" ] && [ "${GITHUB_TOKEN}" != "" ] &&\
```

Comment:
most of these ifs are already taken care of when specifying `on: branch: master` in .travis.yml